docs/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,71 @@
+## Checklist
+<!--
+Mark an `[x]` for completed items.
+-->
+
+* [ ] Update CHANGELOG.rst for each commit separately:
+  * Pay attention to write entries under the "Unreleased" section.
+  * Mark all breaking changes as "**BACKWARD INCOMPATIBLE:**" and put them
+    before non-breaking changes.
+  * If a commit modifies a feature listed under "Unreleased" section,
+    it might be sufficient to modify the existing CHANGELOG entry from previous
+    commit(s). When making a new CHANGELOG entry, augment it.
```

Comment:
Remove "When making a new CHANGELOG entry, augment it." sentence. It is obsolete now.